ocfweb/docs/docs/staff/backend/switch.md

```diff
@@ -1,15 +1,22 @@
 [[!meta title="Managed switches"]]
 
-We use an [Arista 7050SX-64][primary-switch] 10GbE switch as a primary switch
-for our servers, and two [Arista 7048T-A][secondary-switch] 1GbE switches for
+We use an [Arista 7050S-64][primary-switch] 10GbE switch as a primary switch
+for our servers, and two [Arista 7048T-4S][secondary-switch] 1GbE switches for
 desktops and management respectively. These devices were donated to us by
-Arista Networks in Fall 2018. Each device in the lab connects first to the back
+Arista Networks in Fall 2018. 
+
+The Arista 7050S-64 is a Trident+ based switching platform, featuring 48 10G-mode
+ports, and 4 40G-mode ports. Each of the SFP+ cages on the 7050S-64 can support 10G and 1G mode operation.  
```

Comment:
nit: line wrap (and same with the lines after this)

Reply:
line wrap is effort~

I'll play with it though.

Reply:
`gq` if in vim

Reply:
What's it on the github editor on iPhone :P